llvm/lib/Analysis/Loads.cpp

```diff
@@ -334,9 +342,11 @@ bool llvm::isDereferenceableAndAlignedInLoop(
 
   Value *Base = nullptr;
   APInt AccessSize;
+  const SCEV *AccessSizeSCEV = nullptr;
```

Comment:
This seems to be unused

Reply:
Replaced the PtrDiff use below with `AccessSizeSCEV`, thanks